docs/source/getting_started/active_learning.rst

```diff
@@ -0,0 +1,164 @@
+
+   # we want an initial pool of 100 images
+   config = SamplerConfig(n_samples=100, method=SamplingMethod.CORESET, name='initial-selection')
+   initial_selection = al_agent.query(sampler_config)
+
+   assert len(initial_selection) == 100
+
+The query returns the list of filenames corresponding to the initial selection. Additionally, you
+will find that a tag has been created in the web-app under the name "initial-selection".
+Head there to scroll through the samples and download the selected images before annotating them.
+
+
+Active Learning Step
+~~~~~~~~~~~~~~~~~~~~~~
+
+After you have annotated your initial selection of images, you can train a model
+on them. The trained model can then be used to figure out, with which images it 
+has problems. These images can then be added to the labeled dataset.
+
+To do active learning with Lightly, you will need the `ApiWorkflowClient` and `ActiveLearningAgent` from before.
+If you have to re-initialize them, make sure to set the `pre_selected_tag_name` to your
+current selection (if this is the first iteration, this is the name you have passed 
+to the sampler config when doing the initial selection). Note, that if you don't 
+have to re-initialize them, the tracking of the tags is taken care of for you.
+
+.. code-block:: Python
+
+   # re-initializing the ApiWorkflowClient and ActiveLearningAgent
+    api_client = ApiWorkflowClient(dataset_id='xyz', token='123')
+    al_agent = ActiveLearningAgent(api_client, preselected_tag_name='initial-selection')
+
+The next part is what differentiates active learning from simple subsampling; the
+trained model is used to get predictions on the unlabeled data and the sampler then
+decides based on these predictions. To get a list of all filenames in the unlabeled set,
+you can simply call
+
+.. code-block:: Python
+
+   # get all filenames in the unlabeled set
+   unlabeled_set = al_agent.unlabeled_set
+
+Use this list to get predictions on the unlabeled images.
+
+**Important:** The predictions need to be in the same order as the filenames in the
+list returned by the `ActiveLearningAgent` and they need to be stored in a numpy array.
+
+Once you have the scores in the right order, make sure to normalize them such that
+the rows sum to one. Then, create a scorer object like so:
+
+
+.. code-block:: Python
+
+    from lightly.active_learning.scorers import ScorerClassification
+
+    scorer = ScorerClassification(predictions)
+
+Now you have everything to get the next batch of images. One important thing to mention
+here is that the argument `n_samples` always refers to the total size of the labeled set.
+
+.. code-block:: Python
+
+   # we want a total of 200 images after the first iteration
+   # this time, we use the CORAL sampler and provide a scorer to the query
+   config = SamplerConfig(n_samples=200, method=SamplingMethod.CORAL, name='al-iteration-1')
+   al_iteration_1 = al_agent.query(sampler_config, scorer)
```

Comment:
I would call the variable `al_iteration_1` rather `labeled_set_al_iteration_1`. Perhaps you could also print the first 3 elements of it to make clear that it returns the filenames.

Reply:
Maybe I'll just go with `labeled_set_iteration_1`. What do you think?